arcade/key/__init__.py

```diff
@@ -122,6 +122,14 @@
 F14 = 65483
 F15 = 65484
 F16 = 65485
+F17 = 549755813888
+F18 = 554050781184
+F19 = 558345748480
+F20 = 562640715776
+F21 = 566935683072
+F22 = 571230650368
+F23 = 575525617664
+F24 = 579820584960
```

Comment:
Change the 206 in this file to 214 and the build should pass:
https://github.com/pythonarcade/arcade/blob/2c4fd952223fe86dc8fbba958e90f3103cbd9c99/tests/unit/test_key.py#L2-L5